test/svg-sprite.test.js

```diff
@@ -63,10 +67,8 @@ async function compareSvg2Png(svg, png, expected, diff, done, msg) {
     }
 }
 
-before(done => {
-    rimraf(path.join(__dirname, '../tmp'), () => {
-        done();
-    });
+before(() => {
+    rm(path.join(__dirname, '../tmp'), { force: true, recursive: true });
```

Comment:
It should be switched to sync function, or `done` callback should be provided, of should be switched to async/await.
I prefer callback